EnglishTranslation-master/SKILL.tsv

```diff
@@ -4525,42 +4525,42 @@ SKILL_20150714_004524	Increases the damage dealt on an enemy with [Bounce Shot]
 SKILL_20150714_004546	Enemies affected by [Pointing] have a 5% chance per attribute level to be afflicted with [Fear] for 3 seconds.
 SKILL_20150714_004547	DEF of the enemies that are attacked with [Retrieve] will decrease by 10 per Attribute level.
-SKILL_20150714_004548	Flying-type monsters who fall to the ground with [Snatching] will receive 20% additional damage from Slash and Strike attacks per attribute level.
-SKILL_20150714_004549	Increases the movement speed of your Companion by 5% per attribute level with [Praise].
+SKILL_20150714_004548	Flying-type monsters who fall to the ground due to [Snatching] will receive 20% additional damage from Slash and Strike attacks per attribute level.
+SKILL_20150714_004549	[Praise] increases the movement speed of your Companion by 5% per attribute level.
 SKILL_20150714_004550	Coursing: Mental Breakdown
 SKILL_20150714_004551	Decreases the magic defense of an enemy by affected by [Coursing] by 10 per attribute level.
 SKILL_20150714_004552	Hounding: Confusion
-SKILL_20150714_004553	Enemies discovered during [Hounding] will become confused with a certain chance. Increases the chance by 10% per attribute level.
+SKILL_20150714_004553	Enemies discovered during [Hounding] have a certain probability of becoming confused. Increases the chance by 10% per attribute level.
 SKILL_20150714_004554	Increases the damage dealt on an enemy with [Needle Blow] by 1% per attribute level.
 SKILL_20150714_004555	Increases the damage dealt on an enemy with [Wugong Gu] by 1% per attribute level.
 SKILL_20150714_004556	Increases the damage dealt on an enemy with [Throw Gu Pot] by 1% per attribute level.
 SKILL_20150714_004557	A character whose [Poison] is removed due to [Detoxify] will be immune to [Poison] for 2 secs per Attribute level.
 SKILL_20150714_004558	Wugong Gu: Continuous Infection
-SKILL_20150714_004559	Duration of the infected status of the object which is infected by the host of [Wugong Gu] will increase by 1 sec per Attribute level.
-SKILL_20150714_004560	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will last for 2 secs per Attribute level.
-SKILL_20150714_004561	The Max amount of Poison that could be stored increases by 10 per Attribute level with [Poison Pot].
+SKILL_20150714_004559	Duration of the [Wugong Gu] status will increase by 1 sec per Attribute level.
+SKILL_20150714_004560	Even if an enemy steps out of [Throw Gu Pot], [Poison] will last for 2 secs per Attribute level.
+SKILL_20150714_004561	The max amount of Poison that can be stored in the [Poison Pot] increases by 10 per Attribute level.
 SKILL_20150714_004562	Jincan Gu: Vitality
-SKILL_20150714_004563	Life time of the bugs that are born from [Jincan Gu] with a fixed probability will be doubled. The probability will increase by 5% per Attribute level.
+SKILL_20150714_004563	The life time of the bugs created by [Jincan Gu] will be doubled by a fixed chance. The probability will increase by 5% per Attribute level.
```

Comment:
`The probability will increase by 5% per Attribute level.` -> `Increases the chance by 5% per attribute level.`